applications/call_inspector/src/parsers/ci_parser_kamailio.erl

```diff
@@ -282,7 +282,7 @@ cleanse_data_and_get_timestamp(Data0) ->
 -spec cleanse_data_fold({ne_binary() | kz_now()} | ne_binary()
                        ,cleanse_acc()
                        ) -> cleanse_acc().
-cleanse_data_fold({RawTimestamp}, {Acc, TS}) ->
+cleanse_data_fold({'timestamp', RawTimestamp}, {Acc, TS}) ->
```

Comment:
Your check is too naive: this is not a JObj. BTW spec needs to be updated with your changes.